pkg/virt-handler/vm.go

```diff
@@ -756,14 +756,16 @@ func (d *VirtualMachineController) defaultExecute(key string,
 	// set true to ensure that no updates to the current VirtualMachineInstance state will occur
 	forceIgnoreSync := false
 
-	log.Log.V(3).Infof("Processing vmi %v, existing: %v\n", vmi.Name, vmiExists)
+	log.Log.Infof("Processing event %v", key)
 	if vmiExists {
-		log.Log.V(3).Infof("vmi is in phase: %v\n", vmi.Status.Phase)
+		log.Log.Object(vmi).Infof("VMI is in phase: %v\n", vmi.Status.Phase)
+	} else {
+		log.Log.Info("VMI does not exist")
 	}
-
-	log.Log.V(3).Infof("Domain: existing: %v\n", domainExists)
 	if domainExists {
-		log.Log.V(3).Infof("Domain status: %v, reason: %v\n", domain.Status.Status, domain.Status.Reason)
+		log.Log.Object(domain).Infof("Domain status: %v, reason: %v\n", domain.Status.Status, domain.Status.Reason)
+	} else {
+		log.Log.Info("Domain does not exist")
```

Comment:
ditto

Reply:
I think it could (and will) happen the first time a vmi creation is handled and then the handler performs the SyncVMI call which in result creates the domain and generates another event to be processed, and it's part of the normal workflow so info is fine.

Reply:
If it's normal - probably this log will provide only a noise in logs, so maybe it should be logged only on some non standard level (like previously used `V(3)`?)

Reply:
It's normal the first time a vm gets created, not normal after that. 
I think the log is useful to diagnose exceptional events as those I described in https://github.com/kubevirt/kubevirt/issues/2672 
I would certainly not hide it behind a non standard level.
On top of that, the occourrence of these logs are reasonable enough to keep them in order to help us in bad situations.